Automattic/jetpack
# Pay with PayPal Block: Missing quantity input arrows, fixed quantity placeholder.

Now we show the up and down arrows on the quantity input field in Chrome. I've tested in Safari and Firefox, they show the input arrows by default.
Fixed a bug that was causing the quantity field to be displayed incorrectly in the placeholder view.





Fixes #17639

#### Changes proposed in this Pull Request:

* Added a CSS style to show the up and down arrows of the quantity input field of the Pay with PayPal block.
* Changed a CSS property so that the correct padding is applied to the quantity field in the placeholder view while editing a post.

#### Does this pull request change what data or activity we track or use?
It doesn't change.

#### Testing instructions:






1. Apply the patch.
2. Create a new post or edit an existing post.
2. Add a Pay with PayPal block.
3. Enable "Enable people to buy more than one item at a time".
4. Click outside the Pay with PayPal block.
5. The input should be rendered as expected (see images).
6. Save or update the post.
6.  Open the post.
7. The quantity input field should be rendered as expected (see images).

#### Test evidence

##### Placeholder Before
<img width="757" alt="before_placeholder" src="https://user-images.githubusercontent.com/1989914/149366922-dd07ea5a-c296-49f7-b252-7fcc6b7ae3e9.png">

##### Placeholder After
<img width="696" alt="after_placeholder" src="https://user-images.githubusercontent.com/1989914/149367245-9cee632e-50d1-4822-beb3-708f58452f80.png">

##### Post View Before
<img width="748" alt="before_render" src="https://user-images.githubusercontent.com/1989914/149367303-b8d29957-5dcf-482a-a0a3-6126bd621fc8.png">

##### Post View After
<img width="522" alt="after_render" src="https://user-images.githubusercontent.com/1989914/149367330-e5ec46b7-29da-4503-b424-8ba7e811a388.png">
